### PR TITLE
Fixes #8958 - Show tag info for docker images

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

**Comment by ehelms** [comment]
Testing this I see:

manage docker images for repository fedora ssh

**Comment by daviddavis** [comment]
@ehelms fixed.

File: app/views/katello/api/v2/docker_images/show.json.rabl [hunk 1]
```diff
@@ -3,7 +3,8 @@ object @resource
 attributes :uuid => :id
 attributes :size, :image_id
 
-child :tags => :tags do
-  attributes :katello_repository_id => :repository_id
-  attributes :tag
+child :docker_tags => :tags do
+  attributes :repository_id
+  attributes :name => :tag
+  attributes :name
```
Comment (ehelms) [inline app/views/katello/api/v2/docker_images/show.json.rabl]: Why show the same data twice with two different names?

**Comment by ehelms** [comment]
I sync'd this repo - https://registry.hub.docker.com/u/fedora/ssh/ -- should I be seeing a 'latest' tag in the list?

**Comment by daviddavis** [comment]
Try it again.

**Comment by ehelms** [comment]
APJ

**Pull request merged**